common/common.go

```diff
@@ -121,7 +121,7 @@ func AppTreeStoreIDPath(root string, appName types.ACName) string {
 }
 
 // AppImageManifestPath returns the path to the app's ImageManifest file
-func AppInfoImageManifestPath(root string, appName types.ACName) string {
+func AppImageManifestPath(root string, appName types.ACName) string {
```

Comment:
I don't think the Info part is pertinent here. See also `AppTreeStoreIDPath` for example